rollup/core/classes/Rollup.cls

```diff
@@ -1611,6 +1626,9 @@ global without sharing virtual class Rollup {
         metas.remove(index);
         fullRecalcMetas.add(new FullRecalculationMetadata(meta, queryWrapper, calcItemSObjectType));
         prepareReparentDeletionOps(meta, calcItems, oldCalcItems, deleteKeyToReparentOp);
+      } else if (isRefresh) {
```

Comment:
the diff doesn't make this clear, but the if statement above terminates and _then_ this if statement runs